backend/models/cans.py

```diff
@@ -770,10 +769,24 @@ def status(self):
         )
         return can_status
 
+    @property
+    def appropriation_term(self):
+        if self.expiration_date is None or self.appropriation_date is None:
+            return 0
+        return self.expiration_date.year - self.appropriation_date.year
+
```

Comment:
Do we know this should be calendar year and not fiscal year?
Or does the calculation end up the same anyway?

Reply:
I do not know if it's fiscal year or calendar year. That would be a bit harder to calculate because I'd need to define the fiscal year somewhere but it's definitely not impossible.